docs/examples/conversions.md

```diff
@@ -41,15 +41,15 @@ end
 Ruby
 ```ruby
 rule 'Snap Fan to preset percentages' do
-  changed(*CeilingFans)
+  changed CeilingFans.members
   triggered do |item|
     snapped = case item
               when 0...25 then 25
               when 26...66 then 66
-              when 67...100 then 100
+              when 67..100 then 100
```

Comment:
The triple dot was correct here. Can you put it back? 
You want to exclude the ending value.  If it is already at 100 it shouldn't take any action.

Reply:
Ah, didn't read the rest of the rule carefully enough, Will revert it!